packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -172,6 +174,29 @@ export function createResponseState(
       endInlineScript,
     );
   }
+  if (enableFizzExternalRuntime) {
+    if (externalRuntimeConfig !== undefined) {
+      const src =
+        typeof externalRuntimeConfig === 'string'
+          ? externalRuntimeConfig
+          : externalRuntimeConfig.src;
+      const integrity =
+        typeof externalRuntimeConfig === 'string'
+          ? undefined
+          : externalRuntimeConfig.integrity;
+      bootstrapChunks.push(
+        startScriptSrc,
+        stringToChunk(escapeTextForBrowser(src)),
+      );
+      if (integrity) {
+        bootstrapChunks.push(
+          scriptIntegirty,
+          stringToChunk(escapeTextForBrowser(integrity)),
+        );
+      }
+      bootstrapChunks.push(endAsyncScript);
```

Comment:
This will cause the runtime to emit after all the content. The right priority is probably much higher. I just merged support for scripts but we probably want to special case this one as the highest priority script to flush. I'm mostly just flagging it here not so that it gets changed in this PR but that I follow up in my own PR afterwards